base/CMakeLists.txt

```diff
@@ -318,6 +321,8 @@ SET(IP_FILES_H
 	include/FaceDetectsInfo.h
 	include/FaceDetectorXform.h
 	include/TextOverlayXForm.h
+	include/ColorConversionXForm.h
+	include/AbsColorConversionFactory.h
```

Comment:
is this factory required to be exposed via include or should this also be part of src ?

Reply:
Yes , it can be a part of src. Added it to src.

Reply:
Yes it was a mistake , i corrected it.